src/index.js

```diff
@@ -1,10 +1,27 @@
 /**
  * @flow
  */
-import { getThreads } from './store';
+import { getThreads, getMailboxes } from './store';
 
-function renderSidebar() {
-  const messageLIs = getThreads().reduce((html, thread) => `${html} <li>
+/* Called once at the beginning of pageload. In the function, call
+getMailboxes() to find out which ones to render based on what's in the JSON
+file. And since INBOX is the default mailbox to render, start with applying the
+"active" class to that li for styling purposes  */
+function renderNavigation() {
+  const mailboxNames = getMailboxes();
+  const navigationUL = document.querySelector('.client-nav-items');
+  const navItems = mailboxNames.map(name =>
+    name === 'INBOX' ? `<li class="client-nav-item active">${name}</li>` :
+      `<li class="client-nav-item">${name}</li>`).join('');
```

Comment:
Sorry, one more thing. `<li>`s are not interactive elements, so they aren’t accessible to keyboard users by default. Try tabbing to one of these elements to trigger it—you’ll see it’s impossible. That’s how it would be if you used a screenreader, for example.

You could use a natively interactive element like an `<a>` or a `<button>`, or be sure to set `tabindex=0 role=button` and listen for `keypress` of <kbd>Space</kbd> and <kbd>Enter</kbd> in addition to `click`

Reply:
Thanks for pointing this out. I had a feeling it was going to need changing at some point.